test/shared.py

```diff
@@ -0,0 +1,367 @@
+# Copyright 2010-present MongoDB, Inc.
```

Comment:
2024.

Any alternative names for this file? “Shared.py” is pretty generic.

Reply:
What if we combine `test/utils.py` and this new file? Having multiple files that have effectively the same purpose seems unnecessary.

Reply:
Renamed to `test/helpers.py` for now, we can revisit down the road.